cinema/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path
+
+from cinema.views import movie_list, movie_detail
+
+urlpatterns = [
+    path("api/cinema/movies/", movie_list, name="movie-list"),
```

Comment:
imagine you have 50 endpoints and in all you need to write api/cinema/...
it's better to change endpoint in movie_api/urls with `path("api/cinema/", include("cinema.urls", namespace="cinema"))`